### PR TITLE
Change to upd.exe and watchdog2 in inno installer for testing

### PR DESCRIPTION
**Pull request by zanderz**
Here you go @gabriel. Tested building an installer but haven't run it. Note you can build this on the jenkins machine by entering the client branch name in the build with parameters pate.

### CODE REVIEW

**Comment by gabriel** [comment]
trying it out now on jenkins configed with branch

File: packaging/windows/setup_windows_gui.iss [hunk 1]
```diff
@@ -97,10 +98,9 @@ WelcomeLabel2=This will install [name/ver] on your computer.
 
 [Run]
 Filename: "{tmp}\vc_redist.x86.exe"; Parameters: "/quiet /Q:a /c:""msiexec /qb /i vcredist.msi"""; StatusMsg: "Installing VisualStudio 2015 RunTime..."
-Filename: "{app}\{#MyExeName}"; Parameters: "ctl watchdog"; Flags: runasoriginaluser runhidden nowait
+Filename: "{app}\{#MyExeName}"; Parameters: "ctl watchdog2"; Flags: runasoriginaluser runhidden nowait
```
Comment (gabriel) [inline packaging/windows/setup_windows_gui.iss]: run this after dokan install service?

**Comment by gabriel** [comment]
Can we make the watchdog2 log to a `keybase.start.log` file (that file name is convention we use and sent on log send):

`keybase --log-file=%APPDATA%/Keybase/keybase.start.log --log-format=file -d ctl watchdog2`

yes the args need to come before ctl watchdog2 since they are "global" options

**Comment by gabriel** [comment]
Needed to merge with master to get latest stuff

**Comment by zanderz** [comment]
OK check it

**Comment by gabriel** [comment]
lg2m

**Pull request closed without merging**